### PR TITLE
Re-run CVTArchive benchmarks and update CVTArchive

### PR DESCRIPTION
**Pull request by btjanaka**
## Description

See HISTORY.md

## TODO

- [x] Merge #221 (dependency of this PR)

## Status

- [x] I have read the guidelines in [CONTRIBUTING.md](https://github.com/icaros-usc/pyribs/blob/master/CONTRIBUTING.md)
- [x] I have formatted my code using `yapf`
- [x] I have tested my code by running `pytest`
- [x] I have linted my code with `pylint`
- [x] I have added a one-line description of my change to the changelog in `HISTORY.md`
- [x] This PR is ready to go

### CODE REVIEW

**Review by itsdawei — APPROVED** [review]
For the `cvt_add_plot.png`, is it in log scale? if so we might want to mention that in the y-label.

File: ribs/archives/_cvt_archive.py [hunk 1]
```diff
@@ -11,68 +11,67 @@
...
-    Note the logarithmic scales. This plot was generated on a reasonably modern
-    laptop.
+    Tesselation (CVT) to divide an n-dimensional measure space into k cells.
+    The CVT is created by sampling points uniformly from the n-dimensional
+    measure space and using k-means clustering to identify k centroids. When
+    items are inserted into the archive, we identify their cell by identifying
+    the closest centroid in measure space (using Euclidean distance). For
+    k-means clustering, we use :func:`sklearn.cluster.k_means`.
+
+    By default, finding the closest centroid is done in roughly O(log cells)
```
[the remark this answers is not in the document]
Comment (btjanaka) [reply ribs/archives/_cvt_archive.py:26]: Done

File: ribs/archives/_cvt_archive.py [hunk 2]
```diff
@@ -11,68 +11,67 @@
...
+    time using :class:`scipy.spatial.cKDTree`. To switch to brute force, which
+    takes O(cells) time, pass ``use_kd_tree=False``.
```
[the remark this answers is not in the document]
Comment (btjanaka) [reply ribs/archives/_cvt_archive.py:28]: Done

**Comment by btjanaka** [comment]
> For the `cvt_add_plot.png`, is it in log scale? if so we might want to mention that in the y-label.

Noted
